src/dev/flang/ast/AbstractType.java

```diff
@@ -653,6 +654,16 @@ public AbstractType actualType(AbstractFeature f, List<AbstractType> actualGener
   }
 
 
+  /**
+   * Is this the type of a type feature, e.g., the type of `(list
+   * i32).type`. false is this is unknown since type was not resolved yet.
```

Comment:
I find this comment to be misleading since `false` can also mean that it is actually known to not be the type of a type feature.

Reply:
Will enhance this comment. Not proud of this code that requires special handling for instances of `Type`.